index.js

```diff
@@ -0,0 +1,12 @@
+const form = document.querySelector('#form');
+const email = document.querySelector('#email-input');
+const errorMsg = document.querySelector('.error-message');
+let state = false;
+form.onsubmit = () => {
+  if (email.value === email.value.toLowerCase()) {
+    state = true;
+  } else {
+    errorMsg.style.display = 'block';
+  }
+  return state;
+};
```

Comment:
- [ ] Make sure the form data for all inputs should be stored in a single object, and saved to a single local storage registry.
```
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```

Reply:
- [ ] Please use the `localStorage.getItem` and `localStorage.setItem` functions to save and retrieve the form data object to the local storage.

Reply:
- [ ] Remember In order for data to be saved to the local storage the data object should be turned into a string using `JSON.stringify`

Reply:
- [ ] In order for data to be used after being retrieved from the local storage the saved string should be turned back into an object using `JSON.parse`